src/index.html

```diff
@@ -7,8 +7,69 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo" >
+        <a href="#home">
+          <img
+          class="logo_img"
+          src="./images/logo.png"
+          alt="Moyo">
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item is-active">
+            <a class="nav__link is-active" href="#apple">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add empty line between multiline siblings
Apply all over the page
```suggestion
          </li>
          
          <li class="nav__item">
```